halo2_proofs/src/plonk/prover.rs

```diff
@@ -704,12 +687,11 @@ pub fn create_proof<
                 .map(|&(column, at)| ProverQuery {
                     point: domain.rotate_omega(*x, at),
                     poly: &pk.fixed_polys[column.index()],
-                    blind: Blind::default(),
                 }),
         )
         .chain(pk.permutation.open(x))
         // We query the h(X) polynomial at x
         .chain(vanishing.open(x));
 
-    multiopen::create_proof(params, rng, transcript, instances).map_err(|_| Error::Opening)
+    multiopen::create_proof(params, transcript, instances).map_err(|_| Error::Opening)
```

Comment:
create_proof calls gwc/prover.rs create_proof, but then I don't see where is the actual shplonk/prover.rs create_proof called?

BTW, what does gwc stand for?

Reply:
Ah, yes, I see, these are alternatives, it's either gwc or shplonk (but currently gwc is used).

Reply:
gwc refer to the PC described by the original PLONK paper (https://eprint.iacr.org/2019/953.pdf)